docs/standard/serverless-architecture/serverless-conclusion/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Key takeaways - Serverless apps architecture, patterns, and Azure implementation
+description: Serverless provides many benefits. Serverless also has its own challenges. Understanding serverless architecture will help you better architect enterprise applications and use serverless benefits where it makes sense.
+author: JEREMYLIKNESS
+ms.author: jeliknes
+ms.date: 5/3/2018
+ms.prod: .net
+ms.technology: dotnet
+ms.topic: article
+---
+# Conclusion
+
+As a summary and key takeaways, the following are the most important conclusions from this guide.
```

Comment:
Delete "As a summary and key takeaways". Say "The following key takeaways are the ..."